extruct/uniform.py

```diff
@@ -1,16 +1,28 @@
 from six.moves.urllib.parse import urlparse, urljoin
 
 
-def _uopengraph(extracted):
+def _uopengraph(extracted, with_og_arr=False):
     out = []
     for obj in extracted:
-        # In order of appearance in the page
```

Comment:
I think this comment still makes sense - could you please restore it, unless there is a reason to remove it?

Reply:
When `with_og_array` is not set, then for a duplicated key we get the first property. However if `with_og_array` is set then we get a reversed list (i.e order of appearance is opposite to that on the page).